DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.2
 Imports:
     cli,
+    coro,
```

Comment:
I wonder if we should keep this in suggests? Or do we expect that most folks will be using the coro interface?

Reply:
Let's discuss below